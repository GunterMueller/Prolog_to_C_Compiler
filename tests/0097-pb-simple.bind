/* test */

#include <unistd.h>
#include <errno.h>

#define aMacro  \
  ok  \   
  not ok


int errno;
double modf(double, /* out */ double *);

char *getcwd(void *, int);
