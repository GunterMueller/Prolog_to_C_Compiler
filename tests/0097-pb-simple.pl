:- include('tmp/0097-pb-simple.pl').

main :-
	errno(E), display(E),nl,
	getcwd(0, 256, P), display(P), nl,
	modf(12.34, X, Y), display(X/Y), nl.

