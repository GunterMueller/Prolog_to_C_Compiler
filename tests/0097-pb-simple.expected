0
/home/felix/prolog
/(12, 0.34)
